nginx/datadog_checks/nginx/nginx.py

```diff
@@ -338,16 +324,15 @@ def _flatten_json(cls, metric_base, val, tags):
         elif isinstance(val, (int, float, long)):
             output.append((metric_base, val, tags, 'gauge'))
 
-        elif isinstance(val, (text_type, str)):
-            if val[-1] == "Z":
-                try:
-                    # In the new Plus API, timestamps are now formatted
-                    # strings, some include microseconds, some don't...
-                    timestamp = fromisoformat(val[:19])
-                except ValueError:
-                    pass
-                else:
-                    output.append((metric_base, int((timestamp - EPOCH).total_seconds()), tags, 'gauge'))
+        elif isinstance(val, (text_type, str)) and val[-1] == "Z":
+            try:
+                # In the new Plus API, timestamps are now formatted
+                # strings, some include microseconds, some don't...
+                timestamp = fromisoformat(val[:19])
+            except ValueError:
+                pass
+            else:
+                output.append((metric_base, int((timestamp - EPOCH).total_seconds()), tags, 'gauge'))
```

Comment:
Let's use https://github.com/DataDog/integrations-core/blob/f2d17f76e587864207f55ffb87d5b9a52aeac562/datadog_checks_base/datadog_checks/base/utils/time.py#L38

Reply:
Doing in separate PR https://github.com/DataDog/integrations-core/pull/10722